### PR TITLE
Math magicians fetch data from API

### PR DESCRIPTION
**Pull request by KAYEMBAJONATHAN**
- Add Create a new component for displaying a quote
- I have Show a loading state
- I have Show an error state
- Add Import the new component into my main component
- The quote shows on the browser

### CODE REVIEW

File: src/Components/calculator.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import './calculator.css';
 import { useState } from 'react';
 import calculators from '../Logic/calculators';
+import Quote from './Quote';
```
Comment (Munsa1) [inline src/Components/calculator.js]: - [ ] Great job implementing the quotes component, however, the requirement is that the `Quotes` component should be imported into the main component in this case the `App.js` and not in the `Calculator` component, kindly fix this
image
[Requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_fetch_data_from_api.md)

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @KAYEMBAJONATHAN, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Used The `useEffect` Hook Correctly :heavy_check_mark: 
Created A Quotes Component :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._
## Note :red_circle: 
- [ ] Kindly fix the broken link to your issues page for this project in the ReadMe file
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/calculator.js (hunk 1 above)
Comment (peterrobert) [reply src/Components/calculator.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @KAYEMBAJONATHAN , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**